_pages/fsa/install/fsa-schematics.md

```diff
@@ -54,7 +54,7 @@ The following is a description of the various options you can use with the `ng a
 To have a clear picture on how some of these properties can be used, user can enter the following command that sets baseSite(s), currency(ies), and language(s):
 
 ```shell
-ng add @spartacus/fsa-schematics --baseSite=sample-financial-site --currency=usd,eur --language=en,de,fr
+ng add @spartacus/fsa-schematics --base-site=sample-financial-site --currency=usd,eur --language=en,de,fr
```

Comment:
In this file, there seems to be what looks at first sight like a duplicate of the documentation of the schematics options.  We can either update those or see if we can remove them and point to the core/shared Spartacus schematics doc.